new-dti-website/src/app/assets/hero.css

```diff
@@ -0,0 +1,135 @@
+.hero-section {
```

Comment:
Question: Is this CSS file being used on the page? Just wondering if it can be inline with Tailwind, or if we have to keep it in a separate file like this.

Reply:
Sorry 'bout that- it's not being used anywhere! I'll delete the file to minimize any clutter/confusion.